react/react1/week2/todo-list/src/components/Todo.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import todos from "../todos"
+function App() {
```

Comment:
Always the name of the file should be the same as the component's name, so you should change the `App` to `Todo`. The naming also can be abit improved here, since this component is returning our todo list, it's better to call it `TodoList`

Reply:
I have updated name.